src/StreamStats.jl

```diff
@@ -1,15 +1,15 @@
 module StreamStats
     import StatsBase, Distributions
+    import Base.mean, Base.std, Base.var, Base.cov, Base.cor
```

Comment:
I don't understand your comment. Can you give me an example of how I should
be doing this?

On Thursday, February 12, 2015, John Myles White <notifications@github.com>
wrote:

> In src/StreamStats.jl
> <https://github.com/johnmyleswhite/StreamStats.jl/pull/12#discussion_r24590063>
> :
>
> > @@ -1,15 +1,15 @@
> >  module StreamStats
> >      import StatsBase, Distributions
> > +    import Base.mean, Base.std, Base.var, Base.cov, Base.cor
>
> This is just stylistic, but I really like annotating each specific method
> addition for mean as Base.mean. It's more verbose at the definition site,
> but that verbosity really improve local readability of code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/johnmyleswhite/StreamStats.jl/pull/12/files#r24590063>
> .
>